metadata_service/proxy/atlas_proxy.py

```diff
@@ -372,6 +377,24 @@ def _get_reports(self, guids: List[str]) -> List[ResourceReport]:
 
         return parsed_reports
 
+    def _get_owners(self, data_owner: list, fallback_owner: str) -> List[User]:
+        data_owners = list()
```

Comment:
could be renamed to `result` (based on change data_owner > data_owners in method args)

Reply:
changed to `owners_detail`